src/App.css

```diff
@@ -9,12 +9,12 @@
 
 @media (prefers-reduced-motion: no-preference) {
   .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+    animation: App-logo-spin infinite 20s alternate-reverse;
   }
 }
 
 .App-header {
-  background-color: #282c34;
+  background-color: red;
```

Comment:
don't like it, change to black

Reply:
I've changed it to look like this

![image](https://user-images.githubusercontent.com/4099066/143520884-21470f3b-b047-47a6-853f-adb451ca15ae.png)